src/2025h2/reference-expansion.md

```diff
@@ -0,0 +1,147 @@
+multiple areas of Rust, and in several cases, directly worked on the portions
+of Rust that need documenting.
+
+We're expecting to coordinate contributions of new reference material from
+@lcnr, @jackh726, @Amanieu, @GuillaumeGomez, and @yaahc.
+
+Additional people, who may provide review, mentoring, support, and drafting,
+include @joshtriplett, @lcnr, @jackh726, @Amanieu, @GuillaumeGomez, @m-ou-se,
+@petrochenkov, @yaahc, @lucarlig, @midiareshadi, @borsakv, and @SparrowLii.
+
+### The "shiny future" we are working towards
+
+We hope to continue providing ongoing support for reference updates in our
+areas of expertise, as well as making it easier for others to make ongoing
+contributions by providing a starting point. While we don't expect the
+Reference to ever be "done" (much as the Rust language will not be), we hope to
+substantially close the gap between the Reference and the language, 
+
+In the future, we hope that the reference will include full and complete
+documentation for all parts of the Rust language, including the borrow checker
+and the operational semantics.
+
+## Design axioms
+
+The following [design axioms][da] apply:
+* Some documentation may be better suited for the rustc-dev-guide rather than the Rust Reference. If we find ourselves with material better suited for the rustc-dev-guide, we can submit it there, and submit appropriate user-focused subsets of it to the Rust Reference.
+* Behavior of Rust being documented in the Rust Reference does not prevent it from changing in the future. Documenting behavior in the Reference does not inherently change a two-way door to a one-way door, though it *can* contribute to people relying on behavior or to behavior otherwise ossifying.
+  * While we do not want to over-constrain the future evolution of Rust, if we do unintentionally document behavior we don't want to specify or require, we haven't necessarily created a *new* stability guarantee by doing so, and we can still evolve Rust and fix bugs.
+  * Attempting to document something may result in discovering issues with the way it works. If that happens, report it, but don't let attempting to fix it block documenting its current behavior (potentially with appropriate disclaimers to support future changes).
+* We should be explicit about any material that is primarily descriptive rather than normative.
+* We should keep an eye out for potential process improvements that may make it easier to maintain the Reference in the future.
+  * Jack Huey (@jackh726) will work with T-types and T-compiler on potential process improvements to loop in the Reference when making relevant changes as part of those teams.
+
+[da]: ../about/design_axioms.md
+
+## Ownership and team asks
+
+**Owner:** @joshtriplett, in his capacity as `t-lang` and `t-spec` team member,
+will lead this project goal.
+
+Specific task owners for reference chapters are advance predictions, but people may end up trading off on work.
```

Comment:
what does this mean
> people may end up trading off on work

Reply:
@tshepang That's an artifact from an earlier draft that attempted to provide a comprehensive list of which person was going to work on which part of the reference. I'll remove that line.